library/src/main/java/com/getstream/sdk/chat/view/MessageListView.java

```diff
@@ -557,4 +667,74 @@ public void setOnStartThreadListener(Function1<Message, Unit> onStartThreadListe
 
         Drawable getDrawableForAttachmentDescription(Message message, Boolean mine, List<MessageViewHolderFactory.Position> positions);
     }
-}
\ No newline at end of file
+
+    public interface ScrollButtonBehaviour {
+
+        void userScrolledUp();
+
+        void userScrolledToTheBottom();
+
+        void unreadMessages(int count);
```

Comment:
How about something with a verb for this method name? `onUnreadMessageCountChanged`, `updateUnreadMessageCount`, something like that?